atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/benchmarks/table/WideRowsTable.java

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright 2016 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.performance.benchmarks.table;
+
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.TearDown;
+
+import com.google.common.collect.Sets;
+import com.google.common.primitives.Ints;
+import com.palantir.atlasdb.keyvalue.api.Cell;
+import com.palantir.atlasdb.keyvalue.api.KeyValueService;
+import com.palantir.atlasdb.keyvalue.api.TableReference;
+import com.palantir.atlasdb.performance.backend.AtlasDbServicesConnector;
+import com.palantir.atlasdb.performance.benchmarks.Benchmarks;
+import com.palantir.atlasdb.services.AtlasDbServices;
+import com.palantir.atlasdb.transaction.api.TransactionManager;
+
+/**
+ * State class for creating a single Atlas table with one wide row.
+ */
+@State(Scope.Benchmark)
+public class WideRowsTable {
+    public static final int NUM_ROWS = 10000;
+    public static final int NUM_COLS_PER_ROW = 20;
```

Comment:
Any way to get less code duplication between this and WideRowTable (some form of them both being a WideRowsTable, but with different arguments)?  I know this is a little weird because of JMH restrictions (it needs to have complete control of the state object instantiation).  Since JMH makes composition hard we've defaulted to inheritance (see ConsecutiveNarrowTable for an example in existing code).  We're definitely open to other clever ideas to get around JMH.  Happy to talk more in person.